### ISSUE TITLE
No functionality

### ISSUE THREAD

**Issue by richardmatthias** [yi-ji/NeteaseMusicAbroad#2]
Mac OS 10.11.6, Python 2.7.10, NeteaseMusic version 1.5.8

Songs are still greyed out. Moreover, their message signify that they're blocked by region not by other issues.

Another issue is that music.163.com is not accessible on Chrome during the runtime of the executable (other websites are ok)

**Comment by yi-ji** [comment]
Did Chrome show the failure reason as ERR_PROXY_CONNECTION_FAILED? If so, I guess the problem is that the python script crashed.

So please run `python NeteaseMusicProxy.py` and check the outputs. 
Also please make sure port 32794 is not taken on your macbook. If it is, you can change the port number in line 223 of `NeteaseMusicProxy.py`.

**Comment by richardmatthias** [comment]
~/NeteaseMusicAbroad$ python NeteaseMusicProxy.py 
Traceback (most recent call last):
  File "NeteaseMusicProxy.py", line 1, in <module>
    from twisted.web import proxy, http
  File "/Library/Python/2.7/site-packages/twisted/web/proxy.py", line 24, in <module>
    from twisted.internet import reactor
  File "/Library/Python/2.7/site-packages/twisted/internet/reactor.py", line 38, in <module>
    from twisted.internet import default
  File "/Library/Python/2.7/site-packages/twisted/internet/default.py", line 56, in <module>
    install = _getInstallFunction(platform)
  File "/Library/Python/2.7/site-packages/twisted/internet/default.py", line 50, in _getInstallFunction
    from twisted.internet.selectreactor import install
  File "/Library/Python/2.7/site-packages/twisted/internet/selectreactor.py", line 18, in <module>
    from twisted.internet import posixbase
  File "/Library/Python/2.7/site-packages/twisted/internet/posixbase.py", line 18, in <module>
    from twisted.internet import error, udp, tcp
  File "/Library/Python/2.7/site-packages/twisted/internet/tcp.py", line 28, in <module>
    from twisted.internet._newtls import (
  File "/Library/Python/2.7/site-packages/twisted/internet/_newtls.py", line 21, in <module>
    from twisted.protocols.tls import TLSMemoryBIOFactory, TLSMemoryBIOProtocol
  File "/Library/Python/2.7/site-packages/twisted/protocols/tls.py", line 63, in <module>
    from twisted.internet._sslverify import _setAcceptableProtocols
  File "/Library/Python/2.7/site-packages/twisted/internet/_sslverify.py", line 38, in <module>
    TLSVersion.TLSv1_1: SSL.OP_NO_TLSv1_1,
AttributeError: 'module' object has no attribute 'OP_NO_TLSv1_1'

Seems like a Twisted issue?
Though it looks like I've already installed Twisted on my computer since
~/NeteaseMusicAbroad$ pip install Twisted
Requirement already satisfied: Twisted in /Library/Python/2.7/site-packages
Requirement already satisfied: zope.interface>=3.6.0 in /System/Library/Frameworks/Python.framework/Versions/2.7/Extras/lib/python (from Twisted)
Requirement already satisfied: constantly>=15.1 in /Library/Python/2.7/site-packages (from Twisted)
Requirement already satisfied: incremental>=16.10.1 in /Library/Python/2.7/site-packages (from Twisted)
Requirement already satisfied: Automat>=0.3.0 in /Library/Python/2.7/site-packages (from Twisted)
Requirement already satisfied: hyperlink>=17.1.1 in /Library/Python/2.7/site-packages (from Twisted)
Requirement already satisfied: setuptools in /System/Library/Frameworks/Python.framework/Versions/2.7/Extras/lib/python (from zope.interface>=3.6.0->Twisted)
Requirement already satisfied: six in /System/Library/Frameworks/Python.framework/Versions/2.7/Extras/lib/python (from Automat>=0.3.0->Twisted)
Requirement already satisfied: attrs in /Library/Python/2.7/site-packages (from Automat>=0.3.0->Twisted)

**Comment by richardmatthias** [comment]
So after looking at the logs, the following solution fixed it for me:

sudo pip install Twisted==16.4.1
sudo pip install requests

Thanks for your awesome tool and your time.
